js/portofolio.js

```diff
@@ -12,6 +109,91 @@ function close() {
   menuItem.classList.remove('visible_nav_menu');
   menuItem.classList.toggle('hidden');
 }
+
+function closepopup() {
+  modal.classList.remove('visible_modal');
+  modal.classList.toggle('hidden_modal');
+}
+
+function showpopup(index) {
+  let modalcontent = `${'<div class="modal_popup">'
+            + '<div class="modal_container">'
+              + '<div class="modal_close_button" onclick="closepopup()"><i class="fa-solid fa-xmark "></i></div>'
+            + '<div class="modal_image"  onclick="closepopup()"> </div>'
+            + '<div class="desktop_title">'
+              + '<h2 class="modal_title">'}${dataSource[index].card_popup_title}</h2>`
+            + '<div class="buttons for_desktop">'
+              + `<a href="${dataSource[index].card_live_link}"><button class="left_button modal_button">See live <img src="assets/see_live_icon.svg" alt="see live icon"> </button></a>`
+              + `<a href="${dataSource[index].card_source_link}"><button class="left_button modal_button">See Source <img src="assets/github_icon.svg" alt="github icon"> </button></a>`
+            + '</div></div><ul class="modal_popup_technologies">';
+  dataSource[index].card_technologies.forEach((element) => {
+    modalcontent += `<li class="work_language_item">${element}</li>`;
+  });
```

Comment:
- [ ] For the desktop screen, Please make sure the image in the model display in full size, and your design should match with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/189310778-34a6abf4-3ab5-4c23-a156-878c308627bc.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/189310731-d7f022bc-4ed8-4507-92ec-56f489173f62.png)